sdk/react/widgets/VideoViewPlayPause.js

```diff
@@ -133,19 +146,52 @@ class VideoViewPlayPause extends React.Component {
     this.state.play.animationOpacity.setValue(0);
   };
 
+  _renderPlayPauseButton = () => {
+    if (this.state[PLAY].animationOpacity._value === 0) {
+      return this._renderButton(PAUSE);
+    }
+    return this._renderButton(PLAY);
+  };
+
   _renderButton = (name) => {
     const fontStyle = {fontSize: this.props.fontSize, fontFamily: this.props.icons[name].fontFamily};
     const opacity = {opacity: this.state[name].animationOpacity};
     const animate = {transform: [{scale: this.state[name].animationScale}]};
-    const buttonColor = {color: this.props.buttonColor == null? "white": this.props.buttonColor};
+    const buttonColor = {color: this.props.buttonColor == null ? "white" : this.props.buttonColor};
 
     return (
-      <View accessible={false} style={[styles.buttonTextContainer]}>
-        <Animated.Text accessible={false}
-          style={[styles.buttonTextStyle, fontStyle, buttonColor, this.props.buttonStyle, animate, opacity]}>
-          {this.props.icons[name].icon}
-        </Animated.Text>
-      </View>
+      <Animated.Text
+        accessible={false}
+        style={[styles.buttonTextStyle, fontStyle, buttonColor, this.props.buttonStyle, animate, opacity]}>
+        {this.props.icons[name].icon}
+      </Animated.Text>
+    );
+  };
+
+  _renderSeekButton = (name, iconScale) => {
+    if (!this.props.showSeekButtons) {
+      return <View/>
```

Comment:
What happens if you return nothing or null?

Reply:
It's ok to return nothing or null here, I just wanted to prevent  mistakes in the future, because for some android versions we could get error if we have null in the view hierarchy.